comp/autoscaling/datadogclient/fx-mock/fx.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
We actually don't need the `fx-mock` folders since mock are created without FX (we could remove this entirely I think).